lib/ui/page/call/widget/round_button.dart

```diff
@@ -89,217 +74,230 @@ class RoundFloatingButton extends StatefulWidget {
-                    if (b) {
-                      _populateOverlay();
-                    } else {
-                      _hintEntry?.remove();
-                      _hintEntry = null;
-                    }
-                  }
-                : null,
-            onTap: widget.onPressed,
-            child: child,
+            onTap: onPressed,
+            child: LayoutBuilder(
+              builder: (context, constraints) {
+                return Center(
+                  child: SizedBox(
+                    width: (constraints.maxWidth / 60) * (icon.width ?? 60),
+                    height: (constraints.maxHeight / 60) * (icon.height ?? 60),
+                    child: Transform.translate(
+                      offset: offset ?? Offset.zero,
+                      child: SvgIcon(icon),
+                    ),
+                  ),
+                );
+              },
+            ),
           ),
         ),
       ),
     );
+  }
+}
 
-    if (widget.border != null) {
-      button = DecoratedBox(
-        position: DecorationPosition.foreground,
-        decoration: BoxDecoration(
-          border: widget.border,
-          shape: BoxShape.circle,
-        ),
-        child: button,
-      );
-    }
+/// [_ButtonCircle] with optional border and tooltip.
+class _IconButton extends StatelessWidget {
+  const _IconButton({
+    required this.icon,
+    required this.color,
+    required this.withBlur,
+    required this.onPressed,
+    required this.offset,
+    required this.border,
+    required this.hint,
+  });
+
+  /// Callback, called when the button is tapped or activated other way.
+  ///
+  /// If this is set to `null`, the button is disabled.
+  final void Function()? onPressed;
+
+  /// [SvgData] to display instead of [asset].
+  final SvgData icon;
+
+  /// Text that will show above the button on a hover.
+  final String? hint;
 
-    final List<Shadow> shadows = [
+  /// [Offset] to apply to the [icon] or [asset].
+  final Offset? offset;
+
+  /// Background color of the button.
+  final Color? color;
+
+  /// Indicator whether the button should have a blur under it or not.
+  final bool withBlur;
+
+  /// Optional [BoxBorder] of this [RoundFloatingButton].
+  final BoxBorder? border;
+
+  @override
+  Widget build(BuildContext context) {
+    final style = Theme.of(context).style;
+
+    Widget button = _ButtonCircle(
+      color: color,
+      withBlur: withBlur,
+      onPressed: onPressed,
+      offset: offset,
+      icon: icon,
+    );
+
+    button = border != null
+        ? DecoratedBox(
+            position: DecorationPosition.foreground,
+            decoration: BoxDecoration(shape: BoxShape.circle, border: border),
+            child: button,
+          )
+        : button;
+
+    if (hint == null || PlatformUtils.isMobile) return button;
+
+    return Tooltip(
+      message: hint!,
+      preferBelow: false,
+      verticalOffset: 38,
+      waitDuration: const Duration(milliseconds: 300),
```

Comment:
This delay feels like a bad UX in my opinion. Feels like application is loading something when hovering over an icon and that's why it doesn't respond that reactively.